### PR TITLE
ST-296: Remove blue heading from Estimated Pay

### PR DESCRIPTION
**Pull request by SeanArmstrong**
- [x] Unit tests passing
- [x] Coverage reached
- [x] Acceptance tests passing
- [x] Reviewed

### CODE REVIEW

File: app/views/incomes/estimatedPay.scala.html [hunk 1]
```diff
@@ -48,10 +48,9 @@
     employerName = Some(employerName),
     gaCustomTitle = None
 ){
-    <div class="section--branded hmrc flush--bottom soft--top">
-
-        <h2>@Messages("tai.estimatedPay.yourPay")</h2>
-        <p class="h1-heading" id="payBeforeTax">£@MoneyPounds(payBeforeTax.getOrElse(0),0).quantity</p>
+    <div>
+        <h2 class="flush--top">@Messages("tai.estimatedPay.yourPay")</h2>
```
Comment (amaalali) [inline app/views/incomes/estimatedPay.scala.html]: We should probably move away from using `Messages` directly and use `messages` which is implicitly passed to the template

File: app/views/incomes/estimatedPay.scala.html [hunk 2]
```diff
@@ -64,25 +63,27 @@ <h2>@Messages("tai.estimatedPay.yourPay")</h2>
...
+                @Messages("tai.estimatedPay.whySame.text")
             }
         }
 
     </div>
 
+    <div class="section">
+
+        @includes.link(
+            copy=Messages("tai.estimatedPay.checkTaxEstimate"),
```
Comment (amaalali) [inline app/views/incomes/estimatedPay.scala.html]: Same comment on use of `Messages` as above.

File: app/views/incomes/editIncome.scala.html [hunk 3]
```diff
@@ -57,8 +57,8 @@ <h2 class="h2-heading">@Messages("tai.incomes.edit.howTo.subtitle")</h2>
         </ul>
     </div>
 
-    <div class="section--branded hmrc">
-        <h2 class="h2-heading">@Messages("tai.incomes.edit.incomeTax.update.heading")</h2>
+    <div>
+        <h2>@Messages("tai.incomes.edit.incomeTax.update.heading")</h2>
```
Comment (amaalali) [inline app/views/incomes/editIncome.scala.html]: Use of `messages`  over `Messages`?

File: app/views/incomes/confirm_save_Income.scala.html [hunk 4]
```diff
@@ -48,7 +48,7 @@
     <div class="grid-row">
         <div class="grid grid-2-3">
             <div class="inner-block">
-                <h2 class="heading-medium heading--underlined flush--top">@Messages("tai.taxYear",s"${Dates.formatDate(TaxYearResolver.startOfCurrentTaxYear)}",s"${Dates.formatDate(TaxYearResolver.endOfCurrentTaxYear)}")</h2>
+                <h2 class="flush--top">@Messages("tai.taxYear",s"${Dates.formatDate(TaxYearResolver.startOfCurrentTaxYear)}",s"${Dates.formatDate(TaxYearResolver.endOfCurrentTaxYear)}")</h2>
```
Comment (amaalali) [inline app/views/incomes/confirm_save_Income.scala.html]: More a discussion on this page which existed previously, but what do you think about the hardcoded things like these dates in the views? Should these dates be passed in with the view model?

**Comment by amaalali** [comment]
Would it be possible to give the file `tai03092018.scss` a better name? Its a bit of a meaningless name at the moment

File: app/views/incomes/confirm_save_Income.scala.html (hunk 4 above)
Comment (SeanArmstrong) [reply app/views/incomes/confirm_save_Income.scala.html]: Its probably fine when its static stuff like this but I don't know if the view should know about TaxYearResolver

**Comment by Froom2** [comment]
This has been QA'd and looks fine.

File: app/views/incomes/confirm_save_Income.scala.html (hunk 4 above)
Comment (amaalali) [reply app/views/incomes/confirm_save_Income.scala.html]: I was thinking along the same lines. The date is something that could easily be passed in

**Pull request merged**